UI/js/login.js

```diff
@@ -0,0 +1,61 @@
+const loginbtn = document.getElementById('loginbtn');
+const success_login = document.getElementById('success-msg-login');
```

Comment:
Your naming convention isn't following the Airbnb guide. Please use the camel casing.